samples/apps/copilot-chat-app/webapi/FunctionLoadingExtensions.cs

```diff
@@ -51,10 +53,13 @@ internal static class FunctionLoadingExtensions
         kernel.ImportSkill(timeSkill, nameof(TimeSkill));
 
         var chatSkill = new ChatSkill(
-            kernel,
-            chatMessageRepository,
-            chatSessionRepository,
-            promptSettings
+            kernel: kernel,
+            chatMessageRepository: chatMessageRepository,
+            chatSessionRepository: chatSessionRepository,
+            promptSettings: promptSettings,
+            plannerFactory: plannerFactory,
```

Comment:
Why not just pass the planner?

Reply:
Good question - This was mostly a decision to have an obvious separate place for the planner creation code (creating a kernel, importing planner-only functions) that's near the other Semantic Kernel setup code in SemanticKernelExtensions.